scripts/cms-any/update-js.php

```diff
@@ -34,6 +34,6 @@
         uses: silverstripe/gha-update-js@v1
 EOT;
 
-if (check_file_exists('package.json')) {
+if (check_file_exists('package.json') && module_name() !== 'silverstripe/supported-module') {
```

Comment:
```suggestion
if (check_file_exists('package.json') && check_file_exists('yarn.lock')) {
```
Anything with a `yarn.lock` we want to keep js up-to-date.
Anything else, we don't need this workflow.

Reply:
Updated